wolfcrypt/src/ed448.c

```diff
@@ -368,13 +368,18 @@ int wc_ed448_sign_msg_ex(const byte* in, word32 inLen, byte* out,
 
         /* step 2: computing R = rB where rB is the scalar multiplication of
            r and B */
-        ge448_scalarmult_base(&R,nonce);
-        ge448_to_bytes(out,&R);
+        if (ret == 0) {
+            ret = ge448_scalarmult_base(&R,nonce);
+        }
+        if (ret == 0) {
+            ge448_to_bytes(out,&R);
+        }
 
         /* step 3: hash R + public key + message getting H(R,A,M) then
            creating S = (r + H(R,A,M)a) mod l */
-
-        ret = ed448_hash_update(key, sha, ed448Ctx, ED448CTX_SIZE);
+        if (ret == 0) {
```

Comment:
Not required.

Reply:
Don't follow. ret could hypothetically be MEMORY_E from ge448_scalarmult_base() agove.

Reply:
```
if (ret == 0) {
    ge448 _to_bytes(out, &r);

    ret = ed448_hash_update(...);
}
```

ge_448_to_bytes call has no return.